lsstbroker/observation.py

```diff
@@ -16,3 +16,7 @@ def __init__(self, time, light_value, color):
         to geneerate a transients time, light-value and color, it will return
         a tuple.
         """
+
+        self.time = time
+        self.light = light_value
+        self.color = color
```

Comment:
Can you change all instances of color to error for now, since that is what the RR Leary data uses?